Makefile-az.mk

```diff
@@ -38,6 +38,18 @@ az-mkacr: az-mkrg ## Create test ACR
 		--sku Basic --admin-enabled -o none
 	az acr login  --name $(AZURE_ACR_NAME)
 
+az-acrimport: ## Imports an image to an acr registry
+	az acr import --name $(AZURE_ACR_NAME) --source "mcr.microsoft.com/oss/kubernetes/pause:3.6" --image "pause:3.6"
+
+az-cleanenv: 
+	kubectl delete nodepools --all
```

Comment:
Feels like this should be deleted after `nodeclaims`?

I'd order in dependency:
- pods
- nodeclaims
- nodepools
- aksnodeclasses